keith-vscode/src/table/actions.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ *
+ * Copyright 2022 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+interface Action {
```

Comment:
What kind of an action is this? Only used for the table view?
As this should be public API for reusing table views for different use cases, this should be fully documented on its behavior and usage. Also applies for most other public API here.